packages/fast-components-styles-msft/src/flipper/index.ts

```diff
@@ -31,52 +29,49 @@ const styles: ComponentStyles<FlipperClassNameContract, DesignSystem> = (
 
     return {
         flipper: {
-            width: "40px",
-            height: "40px",
+            width: toPx(
+                (designSystem.baseHeightMultiplier + 2) * designSystem.designUnit
+            ),
+            height: toPx(
+                (designSystem.baseHeightMultiplier + 2) * designSystem.designUnit
+            ),
```

Comment:
While these are currently set at 40px, the ( + 2) seems like an arbitrary number to get us to the current size of 40px. I think our choice of `40px` was initially to hit a good middle ground that would work for touch. If we are driving a big part of the touch story through density, could we remove the (+2) for the default here and just have 36px be the new width/height?

Reply:
It would end up being 32px. I do agree that we're addressing touch through density, but it seems common that this control is usually larger than a standard control. Everywhere else we use it (microsoft.com, xbox.com, News) it's already 44px. I'd like to take this an an exception, similar to how badge is always a little smaller.

Reply:
I believe those are 44px though again because they don't have a way to manage density. Our system does - should we consider that? Won't these get significantly larger when density is enabled to support touch?

Reply:
@bheston I think this is still unresolved for me, but I'm not going to block on it as changing the size shouldn't be a breaking change. I would like to have a follow-up on this though